aea/helpers/search/models.py

```diff
@@ -605,7 +609,7 @@ def _check_validity(self):
                 "subexpression must be at least 2.".format(type(self).__name__)
             )
         for constraint in self.constraints:
-            constraint._check_validity()
+            constraint._check_validity()  # pylint: disable=protected-access
```

Comment:
maybe also this one could be made public. As far as I can see, there are no side effects in the implementation of that method.

Reply:
Yeah we can do